documentation/deploy-page.md

```diff
@@ -0,0 +1,28 @@
+# Página de deploy
+
+Por el momento la pagina offline es un cartel estático, que se habilita vía toggle, teniendo MVC offline (habilitando allí la página correspondiente). El formato
+del archivo toggle es el siguiente:
+
+```
+{
+  "status": {
+    "offline": false
+  }
+}
```

Comment:
You can get syntax highlighting:

```suggestion
```json
{
  "status": {
    "offline": false
  }
}
```